src/constructors.jl

```diff
@@ -143,6 +143,7 @@ use their `parent`. Triangular loops aren't yet supported.
 """
 macro avx(q)
     q = macroexpand(__module__, q)
+    isa(q, Expr) || return q
```

Comment:
Can you explain this line?

Reply:
I guess this is a hint to inference that `q` will be an `Expr` from that point on.

Reply:
The result type of `macroexpand` is not inferrable. This just allows the compiler to know that below this line, `q::Expr`.